package.json

```diff
@@ -29,7 +30,12 @@
   "main": "application.js",
   "scripts": {
     "start": "node application.js",
-    "test": "grunt unit"
+    "serve": "grunt serve",
+    "unit": "grunt unit",
+    "coverage-unit": "grunt coverage-unit",
+    "accept": "grunt accept",
+    "coverage-accept": "grunt coverage-unit",
```

Comment:
It does the same as `coverage-unit`, so I think we can delete it from here and from the `README.md` .
I tried to run the grunt coverage-accept task with `grunt coverage-accept` and I got `No coverage information was collected, exit without writing coverage information`, so I think we can ignore it as we don't have coverage information for it.